ui/pages/confirmations/components/confirm/header/header.tsx

```diff
@@ -33,7 +33,16 @@ const Header = () => {
 
   const { currentConfirmation } = useConfirmContext<Confirmation>();
 
-  if (currentConfirmation?.type === TransactionType.tokenMethodTransfer) {
+  const CONFIRMATIONS_WITH_NEW_HEADER = [
```

Comment:
Minor, can this go at the top of the file?

And should we use a more explicit name, what does `New` mean in this context?